### ISSUE TITLE
P5.DOM.js library: Reference needs blue/black 

### ISSUE THREAD

**Issue by kfrajer** [processing/p5.js#3186]
Hi

I was checking the reference for the p5 dom library [here](https://p5js.org/reference/#/libraries/p5.dom).

It says:

> Methods and properties shown in black are part of the p5.js core, items in blue are part of the p5.dom library

However, all the links shown there are blue. Could you correct this please? I am not familiar with the dom library and having this amended will clarify the usage.

Kf

Sorry for the blue highlighting... not my intention

![p5dom-ref-bug](https://user-images.githubusercontent.com/26076840/44965839-0fcb4380-aef5-11e8-9695-8cb2d9a87c54.PNG)

**Comment by lmccart** [comment]
This is correct. All these functions listed are included in the p5.dom.js file, not in the core p5.js library. So if you want to use the, you will need to include that file in your index.html file. By default, the web editor and the example project already include this file.

**Comment by lmccart** [comment]
Oh I see, there are no methods listed in black. So I guess that sentence could be removed completely from the text at the top. That would cause less confusion I think?

**Comment by kfrajer** [comment]
Hi Lauren, I remember this list had some entries in black and some in blue. As described in the preamble of the page, some functions are part of the p5.js core. I am not familiar with the source code to tell them apart.  -Kf
